tests/integration/framework/process/operator/operator.go

```diff
@@ -118,6 +118,10 @@ func (o *Operator) Port() int {
 	return o.port
 }
 
+func (o *Operator) Address() string {
+	return "localhost:" + strconv.Itoa(o.port)
+}
```

Comment:
This is a related change. The const checker failed because `locahost:` was repeated too many times. To resolve this, it has been moved to the relevant processes and exposed as `Address().